src/funfuzz/autobisectjs/autobisectjs.py

```diff
@@ -127,9 +130,9 @@ def parseOpts():  # pylint: disable=invalid-name,missing-docstring,missing-retur
 
     # First check that the testcase is present.
     if "-e 42" not in options.parameters and not Path(options.runtime_params[-1]).expanduser().is_file():
-        print_(flush=True)
-        print_("List of parameters to be passed to the shell is: %s" % " ".join(options.runtime_params), flush=True)
-        print_(flush=True)
+        AUTOBISECTJS_LOG.info("\n")
```

Comment:
`"\n"` will put two lines in the log. You can pass an empty string `..._LOG.info("")` or log a separator like `"=" * 50` if you want this line to be more noticeable. This is a common pattern in this change, so I won't mark them all.

Reply:
I like the `"=" * 50` separator idea, thanks!

Reply:
(well at least for some instances)

Reply:
I'll pass an empty string for least change now, but I'll keep in mind that separator idea for the future.